src/utils/urlUtils.js

```diff
@@ -0,0 +1,18 @@
+export function sanitizeTextFromDomains(text) {
+  if (!text) {
+    return '';
+  }
+  const trimmedText = text.slice(0, 30);
+  const httpRegexG =
+    /[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b[-a-zA-Z0-9()@:%_+.~#?&/=]*/g;
```

Comment:
/[-\w@:%._+~#=]{1,256}\.\w{1,6}\b[-\w()@:%_+.~#?&/=]*/gi;

reason
The \w metacharacter matches word characters.

A word character is a character a-z, A-Z, 0-9, including _ (underscore).

but please check if it works

Reply:
That seems to match it strangely 
<img width="399" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/21a8e9cd-7e10-4631-949e-09191942b1cb">

And this should add the ['scam.io'] in the array so i know what to remove from the string
<img width="403" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/7a243b64-efb0-4ef7-a5ee-392b7c9a70af">

Reply:
`/[-\w@:%.+~#=]{1,64}\.\w{1,6}\b[-\w()@:%+.~#?&/=]*/gi`